### PR TITLE
✨ Tickets: Add channelCreate event for adding users with seeAllTickets

### PR DESCRIPTION
**Pull request by ririxidev**

### CODE REVIEW

**Comment by Bas950** [comment]
Is this tested?

**Comment by ririxidev** [comment]
If you are going to set bot for development mode and change everything sure you will get it tested, no it isnt, at least event on premid server is being executed on new channel. there isnt anything to test since i only map users and use ticket class to change channel perms

File: src/modules/support/events/channelCreate.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { Ticket } from "../classes/Ticket";
+
+const coll = pmdDB.collection("userSettings");
+
+module.exports = async (channel: GuildChannel) => {
+	if (channel.parentID === channels.ticketCategory) {
+		const users = await coll.find({ seeAllTickets: true }).toArray();
+		users.map(async user => {
+			const ticket = new Ticket(),
+				ticketFound = await ticket.fetch("channel", channel.id);
```
Comment (Bas950) [inline src/modules/support/events/channelCreate.ts:14]: just wondering if that is correct :/

Comment (ririxidev) [reply src/modules/support/events/channelCreate.ts:14]: it is, feel free to find same thing in command seealltickets :)

**Pull request merged**